docs/Advanced/Refs.mdx

```diff
@@ -69,14 +69,31 @@ export interface CanvasRef {
   containerHeight?: number;
 
   /**
-   * Center the canvas to the viewport.
+   * Positions the canvas to the viewport.
    */
-  centerCanvas?: () => void;
+  positionCanvas?: (position: CanvasPosition, animated?: boolean) => void;
 
   /**
    * Fit the canvas to the viewport.
    */
-  fitCanvas?: () => void;
+  fitCanvas?: (animated?: boolean) => void;
+
+  /**
+   * Fit a node to the viewport.
+   */
+  fitToNode?: (nodeId: string, animated?: boolean) => void;
```

Comment:
It would be nice if it accepted `string | string[]` so you could center on a group of nodes.

Reply:
ooo yea that sounds good